server/src/internalClusterTest/java/org/opensearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -777,10 +778,10 @@ public void testQueuedSnapshotOperationsAndBrokenRepoOnMasterFailOver() throws E
         createNSnapshots(repoName, randomIntBetween(2, 5));
 
         final long generation = getRepositoryData(repoName).getGenId();
-        final String masterNode = internalCluster().getMasterName();
-        blockNodeOnAnyFiles(repoName, masterNode);
+        final String clusterManagerNode = internalCluster().getMasterName();
+        blockNodeOnAnyFiles(repoName, clusterManagerNode);
         final ActionFuture<CreateSnapshotResponse> snapshotThree = startFullSnapshotFromNonMasterClient(repoName, "snapshot-three");
```

Comment:
`startFullSnapshotFromNonMasterClient` -> `startFullSnapshotFromNonClusterManagerClient`?

Reply:
Changed the method name in the new commit https://github.com/opensearch-project/OpenSearch/pull/2521/commits/87d07e5b81ed67d4f72842ef335ae627839e4954. Thank you for catching it!